test/Microsoft.AspNet.Mvc.FunctionalTests/MvcSampleTests.cs

```diff
@@ -100,7 +100,7 @@ public MvcSampleTests(MvcTestFixture<MvcSample.Web.Startup> fixture)
         {
             // Arrange
             var request = new HttpRequestMessage(HttpMethod.Get, "http://localhost/Home/ReturnUser");
-            request.Headers.Accept.Add(MediaTypeWithQualityHeaderValue.Parse("application/xml;charset=utf-8"));
+            request.Headers.Accept.Add(MediaTypeWithQualityHeaderValue.Parse("application/xml"));
```

Comment:
Because an `Accept` header containing a `charset` is both invalid and _more_ specific than anything a formatter would admit to supporting.